reference/enterprise-api-ref/actions-api.markdown

```diff
@@ -35,3 +35,39 @@ curl -k --user <username>:<password> \
 ```
 HTTP 202 Accepted
 ```
+
+## Trigger agent run
+
+You can trigger an agent run for an individual host.
+
+**URI:** https://hub.cfengine.com/api/actions/agent_run
+
+**Method:** POST
+
+**Parameters:**
+
+* **hostkey** *(string)*
+  Unique host identifier
+
+**Example request (curl):**
+
+```
+curl -k --user <username>:<password> \
+  -X POST \
+  https://hub.cfengine.com/api/actions/agent_run \
+  -H 'content-type: application/json' \
+  -d '{"hostkey": "SHA=hostkey"}'
```

Comment:
I think it would be more clear to include an "actual" hostkey like

```
-d '{"hostkey": "SHA=d9667b668b7d07719f3521346bab80c30e30379ed0ddfb4e3464387b43d7843b"}'
```

Because I might copy the hostkey and do `"SHA=SHA=d9667b668b7d07719f3521346bab80c30e30379ed0ddfb4e3464387b43d7843b"` (aka, SHA= then paste)